scripts/gameBoard.js

```diff
@@ -1,92 +1,88 @@
-let board = [];
-
-function createGameBoard(i, j) {
-  draw.gameBoardFrame(gameBoardWidth,gameBoardHeight,squareSize);
-  for (i=1; i<=gameBoardWidth;i++) {
-    for (j=1; j<=gameBoardHeight;j++) {
-      draw.gameSquare(i,j);
-      //TODO create method to add info the the board object
-      board.push({
-        xCoordinate: i,
-        yCoordinate: j,
-        roomNumber: 0,
-        activeChip: null,
-        bottomChip: null
-      });
-    }
+class GameBoard {
+  constructor() {
+    this.board = [];
+    this.rooms = roomsGame1;
   }
 
-  rooms.forEach(function(el) {
-    el.forEach(function(element) {
-      var x = element[0], y = element[1], roomNumber = element[2];
-      draw.rooms(x,y);
-      (board.find(square => {
-        if(square.xCoordinate === x && square.yCoordinate === y) return true;
-      })).roomNumber = roomNumber;
-    });
-  });
-  console.log(board);
-}
-
-function reloadPage() {
-  window.location.reload(false);
-}
+  createGameBoard() {
+    draw.gameBoardFrame(gameBoardWidth,gameBoardHeight,squareSize);
+    for (var i=1; i<=gameBoardWidth;i++) {
+      for (var j=1; j<=gameBoardHeight;j++) {
+        draw.gameSquare(i,j);
+        //TODO create method to add info the the board object
+        this.board.push({
+          xCoordinate: i,
+          yCoordinate: j,
+          roomNumber: 0,
+          activeChip: null,
+          bottomChip: null
+        });
+      }
+    }
 
-function getClickCoordinates(xClick,yClick) {
-  var boarder = gameBoardFrameSize*2;
-  var xLimit = (boarder+(squareSize*gameBoardWidth)-2);
-  var yLimit = (boarder+(squareSize*gameBoardHeight)-2);
+    this.rooms.forEach(function(el) {
+      el.forEach(function(element) {
+        var x = element[0], y = element[1], roomNumber = element[2];
+        draw.rooms(x,y);
+        (this.getBoard().find(square => {
+          if(square.xCoordinate === x && square.yCoordinate === y) return true;
+        })).roomNumber = roomNumber;
+      }, this);
+    }, this);
```

Comment:
I think we can rewrite this:
this.rooms.forEach(function(el) { ...
to
this.rooms.forEach(el => { ...

And remove the 'this' at the end
}, this);